### PR TITLE
Fix minunit -Wformat-overflow= warning by increasing msg buf size

### PR DESCRIPTION
**Pull request by kazarmy**
**Your checklist for this pull request**
- [X] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [X] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

This pr fixes the following warning (and its ilk) in #260 (https://github.com/rizinorg/rizin/runs/1687264032#step:12:2052 -- also search for `minunit.h`):

dbgopt-minunit

**Test plan**

All builds are green.

**Closing issues**

...

### CODE REVIEW

File: test/unit/minunit.h [hunk 1]
```diff
@@ -168,15 +168,15 @@ void sprint_mem(char *out, const ut8 *buf, size_t len) {
 } while (0)
 
 #define mu_assert_nullable_streq(actual, expected, message) do { \
-		char _meqstr[2048]; \
+		char _meqstr[5120]; \
 		const char *act__ = (actual); \
 		const char *exp__ = (expected); \
 		sprintf(_meqstr, "%s: expected %s, got %s.", (message), (exp__ ? exp__ : "NULL"), (act__ ? act__ : "NULL")); \
```
Comment (wargio) [inline test/unit/minunit.h:174]: how about converting this `sprintf` to `snprintf`?

File: test/unit/minunit.h [hunk 2]
```diff
@@ -168,15 +168,15 @@ void sprint_mem(char *out, const ut8 *buf, size_t len) {
...
 		mu_assert(_meqstr, ((act__) == NULL && (exp__) == NULL) || ((act__) != NULL && (exp__) != NULL && strcmp((exp__), (act__)) == 0)); \
 } while(0)
 
 #define mu_assert_memeq(actual, expected, len, message) do { \
-		char _meqstr[2048]; \
+		char _meqstr[5120]; \
 		const ut8 *act__ = (actual); \
 		const ut8 *exp__ = (expected); \
 		sprintf(_meqstr, "%s: expected ", message); \
```
Comment (wargio) [inline test/unit/minunit.h:182]: same here

**Comment by wargio** [comment]
(same for all `sprintf`)

**Review by ret2libc — APPROVED** [review]
Ok but we can't go beyond this. It's already a lot of stuff on the stack.

File: test/unit/minunit.h (hunk 1 above)
Comment (kazarmy) [reply test/unit/minunit.h:174]: i'll do that in a different pr

**Comment by kazarmy** [comment]
It's only 5k ... 8-bit computers from the 80's have more RAM than that.

**Comment by ret2libc** [comment]
> It's only 5k ... 8-bit computers from the 80's have more RAM than that.

It's not about the RAM a computer has, but the fact that the stack size is limited by the OS (`ulimit -s`) and that in general it is not considered safe allocating more than 1 page on the stack (it may allow to skip the protection page in case of stack overflow...). In any case, considered this is just for unit tests it is probably ok and we do not have to worry about deep stacks.

**Comment by kazarmy** [comment]
Oh sorry didn't know that.

**Pull request merged**